### DISCUSSION TITLE
Cachyos could be supported?

### DISCUSSION THREAD

**Discussion by osevan** [ventoy/Ventoy#1669] [Ideas]
Maybe it will be in future.

**Comment by ventoy** [comment]
Did you test it and it can not work?

**Comment by ph-One** [comment]
It will fail at waiting for a `by-label` mount. I don't know Ventoy or CachyOS well enough to troubleshoot at this time, but as of today CachyOS is very much still "not supported" / does not boot when using Ventoy. I'm interested enough to pursue this in my free time, but someone with more knowledge in this space could likely get it done quicker.

**Comment by GRBL-F** [comment]
for me the text goes by too fast but it says error and says "press any key to continue" then brings me back to the ventoy screen.

**Comment by kalligator** [comment]
Still the "Press any key to continue..." situation happening. 
I don't know if it's happening to only some of us. I tried a couple of previous versions of Ventoy just in case. 
Can anyone confirm it's loading properly for them?

**Comment by steve6375** [comment]
Did you try any boot options such as grub or memory etc.
press f1 to see Ctrl- options
